pkg/operator/resources/syncapi/dashboard.go

```diff
@@ -75,19 +75,22 @@ func addAPIToDashboardObject(dashboard *aws.CloudWatchDashboard, dashboardName s
 		return errors.Wrap(err, fmt.Sprintf("failed to add API \"%s\" to cloudwatch dashboard", apiName))
 	}
 
-	// create widgets for title and metrics
-	apiTitleWidget := aws.TextWidget(1, highestY+1, 22, 1, "## "+apiName)
-	// top left widget
-	statCodeWidget := aws.MetricWidget(1, highestY+2, 11, 6, statusCodeMetric(dashboardName, apiName), "responses per minute", "Sum", 60, config.AWS.Region)
-	// top right widget
-	inFlightWidget := aws.MetricWidget(12, highestY+2, 11, 6, inFlightMetric(dashboardName, apiName), "total in-flight requests", "Sum", 10, config.AWS.Region)
-	// bottem left widget
-	latencyWidgetP50 := aws.MetricWidget(1, highestY+8, 11, 6, latencyMetric(dashboardName, apiName), "median response time (ms)", "p50", 60, config.AWS.Region)
-	// bottom right widget
-	latencyWidgetP99 := aws.MetricWidget(12, highestY+8, 11, 6, latencyMetric(dashboardName, apiName), "p99 response time (ms)", "p99", 60, config.AWS.Region)
+	// create widget for title
+	dashboard.Widgets = append(dashboard.Widgets, aws.TextWidget(1, highestY+1, 22, 1, "## "+apiName))
+
+	var grid aws.CloudWatchWidgetGrid
+	grid.Init(1, highestY+2, 6, 11, 2)
+
+	grid.AddWidget(statusCodeMetric(dashboardName, apiName), "responses per minute", "Sum", 60, config.AWS.Region)
```

Comment:
nit: I'm considering a different layout, what do you think of this one?

https://user-images.githubusercontent.com/808475/86186297-8cc5a500-baed-11ea-885f-d5c301b049eb.png

My reasoning was that the vertical alignment of these could be nice, so your eye can correlate e.g. p99 with p50 response time, and total in-flight requests / avg in-flight requests per replica / active replicas are fairly correlated. It's just a thought, we can discuss.

If you do think this is good, you can grab the image I pasted above for the docs.

Reply:
Totally agree with you, made the changes. Looks better now.